### DISCUSSION TITLE
How to use javascript in Nuxt v3?

### DISCUSSION THREAD

**Discussion by SheikhElMoctarG** [nuxt/nuxt#23316] [Questions]
I dont know, why do the team of nuxt v3 add typescript as the default when create a new project using nuxt? 

how to use javascript in nuxtjs 3??

**Comment by saifobeidat** [comment]
Nuxt support both js and ts files

Just change any `.ts` file to `.js`.and remove the types from your file.

Also in your SFC remove lang="ts"...

Is there any other area you couldn't use plain js?
